### PR TITLE
add dolores.conf

### PR DESCRIPTION
**Pull request by gemfarmer**
As per your comment on the dolores-bot repot (https://github.com/18F/dolores-landingham-bot/issues/2#issuecomment-141159857), I made an nginx .conf file. Am I missing anything?

### CODE REVIEW

**Comment by mbland** [comment]
So there is going to be a staging instance at `dolores-staging.18f.gov`?

Also need to add a line at the bottom of `deploy/etc/nginx/nginx.conf` to include the new config.

**Comment by jessieay** [comment]
I don't think we need staging for this, but don't feel strongly either way.

**Comment by gemfarmer** [comment]
@mbland I added the reference to `dolores.conf` in `nginx.conf` 
I was assuming that we would have a staging site, but if there is a reason why culling that would be better/easier, then we could just have a production site.

File: deploy/etc/nginx/vhosts/dolores.conf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+server {
+  listen 127.0.0.1:8080;
+  server_name dolores.18f.gov;
+  port_in_redirect off;
+
+  location / {
+    resolver 8.8.8.8;
+    set $backend_upstream "https://dolores.18f.gov";
```
Comment (mbland) [inline deploy/etc/nginx/vhosts/dolores.conf]: This should be the Cloud Foundry URL where the app will actually be deployed. (As it stands, it's going to try to proxy to itself.)

File: deploy/etc/nginx/vhosts/dolores.conf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+server {
+  listen 127.0.0.1:8080;
+  server_name dolores-staging.18f.gov;
+  port_in_redirect off;
+
+  location / {
+    resolver 8.8.8.8;
+    set $backend_upstream "https://dolores-staging.18f.gov";
```
Comment (mbland) [inline deploy/etc/nginx/vhosts/dolores.conf]: Same here re: CF URL.

**Comment by mbland** [comment]
@gemfarmer Whether or not you want a separate staging instance is entirely up to you. Can always add one later if you want, but it's probably best to take it out of the config for now if you don't.

I'll set up the DNS mapping for dolores.18f.gov.

**Comment by mbland** [comment]
OK, the DNS mapping is live.

File: deploy/etc/nginx/vhosts/dolores.conf (hunk 1 above)
Comment (gemfarmer) [inline deploy/etc/nginx/vhosts/dolores.conf]: Ok. I'm working on creating the URL. I've been running into some CF issues.

**Comment by mbland** [comment]
This is now running in prod, and all is well... Except there needs to be a Cloud Foundry route from `dolores.18f.gov` to `dolores-app.18f.gov`. Sorry for not recalling the lessons from #393.

**Pull request merged**